tests/angular/partials.spec.js

```diff
@@ -0,0 +1,22 @@
+ngDescribe({
```

Comment:
I'd replace this test with maybe a sample test for the `login-form` component.. what do you think? Because it's guaranteed that everyone will immediately change their footer & it feels weird to assert for the exact same template
I can write the login-form test if you want.. I'd probably just assert that it has a `email` and `password` in its template.. no need to overcomplicate it for now